CHANGELOG.md

```diff
@@ -2,6 +2,40 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-beta.1 November 18, 2021
+
+## Features ✨ and improvements 🏁
+* Remove `MapView` argument when constructing `AnnotationManager`. ([#766](https://github.com/mapbox/mapbox-maps-android/pull/766))
+* Implement cluster API on top of `MapboxMap.queryFeatureExtensions`. ([#773](https://github.com/mapbox/mapbox-maps-android/pull/773))
```

Comment:
Could use some wording what benefits this brings: eg. "this makes using GeoJson Cluster APIs more easy to use as .."